apps/user-office-frontend/src/components/timeout/TimeOutPrompt.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+import React from 'react';
+
+type IdleTimeoutPromptProps = {
+  isIdle: boolean;
+  onConfirm: () => void;
+};
+
+const useStyles = makeStyles((theme) => ({
+  title: {
+    marginTop: '12px',
+    color: theme.palette.error.main,
+  },
+  body: {
+    display: 'grid',
+    placeItems: 'center',
+  },
+}));
+
+function IdleTimeoutPrompt(props: IdleTimeoutPromptProps): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <>
```

Comment:
I think you don't need this React fragment here or empty tag because the Dialog element wraps everything.

Reply:
Yep missed that thanks!